Portal/Helpers/IQueryStringHelper.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Portal.Helpers
+{
+    public interface IQueryStringHelper
+    {
+        string RemoveCultureQueryString(string existingQueryStrings);
+    }
+}
```

Comment:
Put the interface into an Interfaces directory under Helpers

Reply:
Done. Thanks.